daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -422,9 +422,12 @@ runLedgerNavigator flags remainingArguments = do
 
         writeFileUTF8 navigatorConfPath (T.unpack $ navigatorConfig partyDetails)
         unsetEnv "DAML_PROJECT" -- necessary to prevent config contamination
-        withJar damlSdkJar [logbackArg] ("navigator" : navigatorArgs ++ ["-c", confDir </> "ui-backend.conf"]) $ \ph -> do
```

Comment:
Looking at this code as a whole, it doesn't really make sense anymore. This function is doing a bunch of extra work to pass in parties to the navigator (all the stuff with `partyDetails` and `navigatorConfig` and `confDir`, etc) and then it's passing the `ignoreProjectParties` flag.

I think we can just remove all the stuff related to the config / parties, and then just pass in the ignoreProjectParties flag (with the host and port and remainingArguments).

The other thing to improve this code is to pass all the navigator args in `navigatorArgs` instead of adding some here.

Reply:
that's of course true, I'll change it.